styles/styles.css

```diff
@@ -711,6 +914,14 @@ main > .section > .grid-container {
   }
 }
 
+/* ** Display Utilities ** */
+
+.util-hide-at-large {
```

Comment:
nice! will you add this to the section header button? is it the same MQ?

Reply:
It's not the same MQ, but I'll update the button group CSS!